core/src/main/java/hudson/model/ViewGroupMixIn.java

```diff
@@ -163,12 +163,14 @@ public View getView(@CheckForNull String name) {
     }
 
     /**
-     * Returns the primary {@link View} that renders the top-page of Hudson.
+     * Returns the primary {@link View} that renders the top-page of Hudson or
+     * {@code null} if there is no primary one defined.
      */
     @Exported
+    @CheckForNull
```

Comment:
is there anywhere in jenkins core that needs updating to now handle the nullness?

Reply:
Wouldn't that cause a compile error from spotbugs now that the annotation is there?